test/mocha.opts

```
@@ -1,4 +1,3 @@
---bail
```

Can we not commit this change? I think it's useful if tests fail early in the CI.